drivers/dma/dmamux_stm32.c

```diff
@@ -46,21 +46,21 @@ struct dmamux_stm32_config {
 	struct stm32_pclken pclken;
```

Comment:
Same for this file. We're not making any changes to it and thus don't have to care about the formatting. So all changes should be removed.

Reply:
restored original file